### ISSUE TITLE
Option to speak current dialogue bubble immediately upon TTS enable

### ISSUE THREAD

**Issue by ryankhart** [karashiiro/TextToTalk#34]
This is a bit of a nitpick, but I love this plugin so much not to give more feedback.

TextToTalk used to speak the current NPC dialogue that is currently on the screen when I use the keyboard shortcut to enable TTS. This functionality, unfortunately, got removed in [this commit](https://github.com/karashiiro/TextToTalk/commit/781a6fae27e39d2d15cf6c0697221ffd2362b6f1).

Was it sometimes annoying hearing TTS NPC dialogue from an old cutscene I had 15 minutes ago? Sure, I guess, but to me, it's more disappointing that, if I forget to enable TTS before an NPC conversation, then I have to manually read the first line of dialogue before TTS kicks in for the next line of dialogue.

Furthermore, even if I never forget to enable TTS before NPC dialogue, I don't know which NPC dialogue will be voice acted before it starts, so, before this commit, I had previously disabled TTS just in case there was voice acting, and if not, then I'd enable it with the shortcut.

However, if there is imminent functionality to automatically disable TTS during voided cutscenes as described in this [issue](https://github.com/karashiiro/TextToTalk/issues/7), my issue described here is more or less a non-issue, as I would have no reason to ever manually disable TTS.

**Comment by karashiiro** [comment]
This might be a duplicate of #32, waiting on a testing build being merged.

**Comment by ryankhart** [comment]
Oh my bad. I didn't think to check through the closed issues for my issue before making a new one. That sounds exactly like my issue. I'll close this issue for now and wait for the next update.

**Comment by johnysandels** [comment]
I've just tested this and it does in-fact work as described, with some caveats. It only works while  "Cancel the current speech when new text is available or text is advanced" is selected. If you have that disabled, enabling the plugin by toggle while a textbox is open, the textbox that is currently open isn't read.

**Comment by ryankhart** [comment]
I just figured out how to install the testing versions of plugins! I didn't realize that it was right in the GUI settings to enable it.

I just tested all of my use cases, that I'm aware of, and in 1.8, it works great! I still haven't bothered with messing with lexicons yet, though.
